src/components/VideoProvider/index.test.tsx

```diff
@@ -22,29 +22,29 @@ jest.mock('./useHandleTrackPublicationFailed/useHandleTrackPublicationFailed');
 jest.mock('./useHandleOnDisconnect/useHandleOnDisconnect');
 
 describe('the VideoProvider component', () => {
-  // it('should correctly return the Video Context object', () => {
-  //   const wrapper: React.FC = ({ children }) => (
-  //     <VideoProvider onError={() => {}} onDisconnect={mockOnDisconnect} options={{ dominantSpeaker: true }}>
-  //       {children}
-  //     </VideoProvider>
-  //   );
-  //   const { result } = renderHook(useVideoContext, { wrapper });
-  //   expect(result.current).toEqual({
-  //     isConnecting: false,
-  //     localTracks: [{ name: 'mockTrack' }],
-  //     room: mockRoom,
-  //     onError: expect.any(Function),
-  //     onDisconnect: mockOnDisconnect,
-  //     getLocalVideoTrack: expect.any(Function),
-  //   });
-  //   expect(useRoom).toHaveBeenCalledWith([{ name: 'mockTrack' }], expect.any(Function), {
-  //     dominantSpeaker: true,
-  //   });
-  //   expect(useLocalTracks).toHaveBeenCalled();
-  //   expect(useHandleRoomDisconnectionEvents).toHaveBeenCalledWith(mockRoom, expect.any(Function));
-  //   expect(useHandleTrackPublicationFailed).toHaveBeenCalledWith(mockRoom, expect.any(Function));
-  //   expect(useHandleOnDisconnect).toHaveBeenCalledWith(mockRoom, mockOnDisconnect);
-  // });
+  it('should correctly return the Video Context object', () => {
+    const wrapper: React.FC = ({ children }) => (
+      <VideoProvider onError={() => {}} onDisconnect={mockOnDisconnect} options={{ dominantSpeaker: true }}>
+        {children}
+      </VideoProvider>
+    );
+    const { result } = renderHook(useVideoContext, { wrapper });
+    expect(result.current).toEqual({
+      isConnecting: false,
+      localTracks: [{ name: 'mockTrack' }],
+      room: mockRoom,
+      onError: expect.any(Function),
+      onDisconnect: mockOnDisconnect,
+      getLocalVideoTrack: expect.any(Function),
+    });
+    expect(useRoom).toHaveBeenCalledWith([{ name: 'mockTrack' }], expect.any(Function), {
+      dominantSpeaker: true,
+    });
+    expect(useLocalTracks).toHaveBeenCalled();
+    // expect(useHandleRoomDisconnectionEvents).toHaveBeenCalledWith(mockRoom, expect.any(Function));
```

Comment:
I see on the code that _useHandleRoomDisconnectionEvents_ expects 3 arguments, I think it should be:
```suggestion
    // expect(useHandleRoomDisconnectionEvents).toHaveBeenCalledWith(mockRoom, expect.any(Function), expect.any(Function));
```